### ISSUE TITLE
Chrome Profiles, no Default anymore

### ISSUE THREAD

**Issue by pocketnone** [MoistCoder/CockyGrabber#35]
Some Update ago Chrome changed every Directory. 

Chrome created on Default a new Profile and stores the Data inside the Profile
 
Regex: `^Profile [1-999]$`  

Some Paths Changes (Password, Cookies, Bookmarks etc) 

**New Paths:** 
**Local State** -> User Data\\Local State
**Cookies** -> Profile 1\\Network\\Cookies
**Bookmarks** -> Profile 1
**Login Data** -> Profile 1
**Top Sites** -> Profile 1
**History** -> Profile 1

Maybe i Push an Update

**Comment by MoistCoder** [comment]
Nothing changed. "Profile 1" is just the secondary profile, while "Default" is the default profile. You probably have 2 profiles where the standard profile data, created when you set up chrome the first time, is located in the "Default" folder. The data of the second profile, however, which you yourself created, is located in the "Profile 1" folder.

Here you see my chrome profile (In german and censored for privacy reasons):
![profile](https://user-images.githubusercontent.com/77922456/195188473-e891633e-6ee4-4498-8576-7feface51e16.png)


The data of the current one I'm using is located in "Default".
![Default](https://user-images.githubusercontent.com/77922456/195188226-75d76765-f8bc-454c-ad89-ea4e2f3025d8.png)

I created two test profiles for this example, which makes 3 profiles in total. (Without "Gast" meaning guest in german, which is the preinstalled guest profile):
![two-test-profiles](https://user-images.githubusercontent.com/77922456/195188413-8406a53d-d4db-4e04-9d4d-f97f6f783059.png)
`TestProfileABC` and `TestProfileDEF`

The data of TestProfileABC is located in "Profile 1", while the data of TestProfileDEF is located in "Profile 2":
![image](https://user-images.githubusercontent.com/77922456/195189004-c2aecae1-1195-4860-abeb-ea16534aa32f.png)

So again: Chrome didn't create a new profile; you did. No paths were changed. Nothing changed.
CockyGrabber just isn't able to grab from multiple profiles of chromium/blink browsers at the moment (Maybe in a future update tho).
